distribution/Dockerfile

```
@@ -0,0 +1,12 @@
+FROM python:3.12.1-slim-bullseye
+
+WORKDIR /app
+
+COPY ../requirements /tmp/requirements/
```

1. build запускаем из папки уровнем выше, так что ../ не нужно
2. можно оставлять requirements в докере, особенно если будем запускать тесты на локале
```suggestion
COPY requirements /src/requirements/
```